src/pages/settings/Payments/paymentPropTypes.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import CONST from '../../../CONST';
+
+/** Array of bank account objects */
+const bankAccountListPropTypes = PropTypes.arrayOf(PropTypes.shape({
```

Comment:
It seems these are only used once and therefore do not need to be moved into their own file.